gamedaoco/gamedao-haiku
# feat: organization members

## Related to issue #134 

### Done:

- Integrate the members for the organization
- Create a data grid for the members list
- Sort only Name and Role


### To Be Done:
- generate mock data and integrate it from GraphQL

#### Preview:

<img width="1185" alt="Screen Shot 2022-07-07 at 4 23 47 PM" src="https://user-images.githubusercontent.com/67205042/177784334-ade45d93-c1e5-4be8-b2df-5a9c21cc3bc7.png">

### How to Test:

- Navigate to an organization and check the members
- Join an organization and check if you are a member
